share/picongpu/examples/FoilLCT/etc/picongpu/4.cfg

```diff
@@ -72,11 +72,12 @@ TBG_chargeConservation="--chargeConservation.period 100"
 
 # regular output
 TBG_hdf5="--hdf5.period 250 --hdf5.file simData"
```

Comment:
please remove this line if you switch from HDF5 to openPMD to not have unsed stuff in.